src/api/spec/README.md

```diff
@@ -6,7 +6,13 @@ test things based on the following rules:
 * Every main workflow has a feature test
 
 ## Running the spec
-`bundle exec rake spec`
+```bundle exec rspec```
+
+and to run a single test file:
+
+```
+bundle exec rspec spec/models/user_spec.rb
+```
```

Comment:
I had some troubles to get it running today together with vagrant... But I can drop it!